examples/notebooks/get-started/learn_beam_transforms_by_doing.ipynb

```diff
@@ -0,0 +1,740 @@
+        "\n",
+        "Below the code cell you will edit is a hidden answer code cell to check your work. If you're stuck, try opening the hint first!"
+      ],
+      "metadata": {
+        "id": "OJCZKYxGNAfU"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#@title Open code to show the hint\n",
+        "\n",
+        "#Hint: The newly defined transform combines the Count and FlatMap transform.\n",
+        "#Replace the `FlatMap` and `Count` transforms with CountWords() (see above)*italicized text*"
+      ],
+      "metadata": {
+        "id": "9XO-uwhL8w27",
+        "cellView": "form"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#@title EDIT THIS CODE CELL TO USE YOUR `CountWords`\n",
+        "# EDIT THIS CODE CELL\n",
+        "\n",
+        "inputs_pattern = 'data/*'\n",
+        "outputs_prefix = 'outputs/part3'\n",
+        "\n",
+        "# Running locally in the DirectRunner.\n",
+        "with beam.Pipeline() as pipeline:\n",
+        "  word_count = (\n",
+        "      pipeline\n",
+        "        | 'Read lines' >> beam.io.ReadFromText(inputs_pattern)\n",
+        "        | 'Find words' >> beam.FlatMap(lambda line: re.findall(r\"[a-zA-Z']+\", line))\n",
+        "        | 'Count words' >> beam.combiners.Count.PerElement()\n",
+        "        | 'Write results' >> beam.io.WriteToText(outputs_prefix)\n",
+        "  )\n",
+        "\n",
+        "pipeline.run()\n",
+        "# Sample the first 20 results, remember there are no ordering guarantees.\n",
+        "run('head -n 20 {}-00000-of-*'.format(outputs_prefix))"
+      ],
+      "metadata": {
+        "id": "FyW5ug6S3-1B"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#@title Answer\n",
+        "inputs_pattern = 'data/*'\n",
+        "outputs_prefix = 'outputs/part3'\n",
+        "\n",
+        "# Running locally in the DirectRunner.\n",
+        "with beam.Pipeline() as pipeline:\n",
+        "  word_count = (\n",
+        "      pipeline\n",
+        "        | 'Read lines' >> beam.io.ReadFromText(inputs_pattern)\n",
+        "        # The composite transform function you created above\n",
+        "        | 'Count Words' >> CountWords()\n",
+        "        | 'Write results' >> beam.io.WriteToText(outputs_prefix)\n",
+        "  )\n",
+        "\n",
+        "# Sample the first 20 results, remember there are no ordering guarantees.\n",
+        "run('head -n 20 {}-00000-of-*'.format(outputs_prefix))"
+      ],
+      "metadata": {
+        "id": "6Cn4pk3j8K4p",
+        "cellView": "form"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### 3.3 Creating Your Own Composite Transform With `PTransform` Directly"
```

Comment:
This section is wordy IMO. It might make sense to link to documentation here

Reply:
I indeed has the link: https://beam.apache.org/documentation/programming-guide/#composite-transform-creation. But I do not think the link is good for beginners.